vita3k/gui/src/vita3k_update.cpp

```diff
@@ -79,6 +79,7 @@ bool init_vita3k_update(GuiState &gui) {
         std::thread get_commit_desc([dif_from_current]() {
             // Calculate Page and Per Page Get
             std::vector<std::pair<uint32_t, uint32_t>> page_count;
+            page_count.reserve(dif_from_current / 100);
```

Comment:
if you can revert it please, i have pr one totaly change code, and this code is wrong here